src/xcode/ENA/ENAUITests/ExposureDetection/ENAUITests_04b_ExposureDetection.swift

```diff
@@ -33,6 +33,30 @@ class ENAUITests_04b_ExposureDetection: CWATestCase {
 		app.buttons[AccessibilityIdentifiers.SurveyConsent.acceptButton].waitAndTap()
 	}
 
+	func test_navigateToInformationGuideHygiene() throws {
+		launch()
+
+		// Tap risk card and wait for exposure details.
+		app.cells.buttons[AccessibilityIdentifiers.Home.RiskTableViewCell.topContainer].waitAndTap()
+		XCTAssertTrue(app.buttons[AccessibilityIdentifiers.AccessibilityLabel.close].waitForExistence(timeout: .long))
+
+		// Scroll to and tap iButton from cell
+		let iButton = app.cells[AccessibilityIdentifiers.ExposureDetection.detailsGuideHygiene].buttons.firstMatch
+		iButton.waitAndTap()
+	}
+
+	func test_navigateToInformationGuideHome() throws {
```

Comment:
Hmm can't we make these two tests into one? Would safe us the (long) time it takes for the sim to launch to app.

Reply:
I'll check that

Reply:
@marcussc I merged them together - because the AccessibilityIdentifier of the close button gets reused all the time, I added some code to make the one I need while UI testing is in progress.
Please have a second look.